### PR TITLE
Add interactivity to selecting an action in the Actions panel

### PR DESCRIPTION
**Pull request by hatched**
## Done

The actions list is now styled to match the designs.
- Selecting an action opens the actions option list (if one exists)
- If an option description is longer than 30 characters it'll be collapsed and allow for expansion.

Note:
It's still not functional - this is purely UI interactivity. Because of this the functionality tests are going to come in the follow-up branch which hooks this up to the API.

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- View a charm with actions, select an action and see the available interactions

## Details

Fixes https://github.com/canonical-web-and-design/juju-squad/issues/1663

## Screenshots
Screen Shot 2021-03-18 at 3 26 40 PM

heightanim

### CODE REVIEW

File: src/components/RadioInputBox/RadioInputBox.tsx [hunk 1]
```diff
@@ -1,51 +1,155 @@
...
+      // override the height set in css for the collapsed size.
+      wrapper.style.height = endHeight + "px";
+    } else if (wrapper.offsetHeight !== initialHeight) {
+      // Do not animate if the wrapper height is already in the closed state
+      // and this hook runs for the closed state again.
+      endHeight = initialHeight;
+      startHeight = container.offsetHeight + paddingNumber;
+      duration = startHeight;
+    }
+    const animation = wrapper.animate(
```
Comment (hatched) [inline src/components/RadioInputBox/RadioInputBox.tsx:73]: This is using [WAAPI](https://developer.mozilla.org/en-US/docs/Web/API/Web_Animations_API/Web_Animations_API_Concepts) so it's still as performant as doing it in CSS but this actually works 😆

**Review by anthonydillon — CHANGES_REQUESTED** [review]
This looks good. Can you add some tests to the new element. 

There are a bunch of Vanilla patterns that could be used here. Such as required form elements, etc. But they can be cleaned up after.

**Review by anthonydillon — APPROVED** [review]
LGTM :+1:

**Comment by hatched** [comment]
Thanks all

**Pull request merged**